### PR TITLE
Add VS Code devcontainer configuration.

### PR DESCRIPTION
**Pull request by alunduil**
This sets up the repository to be more easily used with VS Code and updates the
scaffolding to be consistent with my current Python project layout.

### CODE REVIEW

**Review by LinoBigatti — APPROVED** [review]
LGTM. Its been a long time since I have contributed to this project. Maybe we should catch up, since I dont remember how much of my code I actually finished.

File: etest/tests.py [hunk 1]
```diff
@@ -41,73 +42,81 @@ def __init__(
...
+        result = {}
 
         if "python" in self.ebuild.compat:
-            e["PYTHON_TARGETS"] = " ".join(self.ebuild.compat["python"])
+            result["PYTHON_TARGETS"] = " ".join(self.ebuild.compat["python"])
 
             # Things still want python2…☹
-            if "python2_7" not in e["PYTHON_TARGETS"]:
-                e["PYTHON_TARGETS"] += " python2_7"
+            if "python2_7" not in result["PYTHON_TARGETS"]:
```
[the remark this answers is not in the document]
Comment (LinoBigatti) [reply etest/tests.py:109]: Marking as resolved as it is only a comment.

**Comment by alunduil** [comment]
@LinoBigatti we should definitely catch up. Hit me up on matrix and we'll sync a bit.

**Comment by LinoBigatti** [comment]
@alunduil sure enough. Its been a while since Ive opened matrix. Sorry if I have missed any messages there.

**Pull request merged**